crates/ruff_formatter/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "ruff_formatter"
+version = "0.0.0"
+edition = "2021"
+authors = ["Rome Tools Developers and Contributors"]
```

Comment:
You may want to change the authors to Ruff.

Reply:
I wasn't certain what the "right" pattern was here. I noticed (e.g.) that `rome_rowan` has `authors = ["Aleksey Kladov <aleksey.kladov@gmail.com>"]`.